docs/pages/installation/_index.md

```diff
@@ -5,20 +5,252 @@ weight: 2
+export GKE_CLUSTER_ZONE=europe-west-1b # the GCP zone where your GKE cluster is executed
+```
+
+First, you should create a GCP Service account. 
+This will provide you an identity used by Liqoctl to query all the information needed to properly configure Liqo on your cluster.
+
+The ServiceAccount can be created using:
+```bash
+gcloud iam service-accounts create ${SERVICE_ACCOUNT_ID} \
+    --description="DESCRIPTION" \
+    --display-name="DISPLAY_NAME" \
+    --project="${PROJECT_ID}"
+```
+
+Second, you should provide the ServiceAccount just created with the rights to inspect the cluster and virtual networks parameters:
+
+```bash
+gcloud projects add-iam-policy-binding ${PROJECT_ID} \
+    --member="serviceAccount:${SERVICE_ACCOUNT_ID}@${PROJECT_ID}.iam.gserviceaccount.com" \
+    --role="roles/container.clusterViewer"
+gcloud projects add-iam-policy-binding ${PROJECT_ID} \
+    --member="serviceAccount:${SERVICE_ACCOUNT_ID}@${PROJECT_ID}.iam.gserviceaccount.com" \
+    --role="roles/compute.networkViewer"
+```
+
+Third, you should create and download valid service accounts keys, as presented [by the official documentation](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys).
+
+The keys will be used by liqoctl to authenticate to GCP as the service account we just created.
+
+```bash
+gcloud iam service-accounts keys create ${SERVICE_ACCOUNT_PATH} \
+    --iam-account=${SERVICE_ACCOUNT_ID}@${PROJECT_ID}.iam.gserviceaccount.com
+```
+
+{{% /tab %}}
+{{% tab name="K3s" %}}
+
+##### Supported CNIs
+
+Liqo supports K3s clusters using the following CNIs:
+* [Weave](https://github.com/weaveworks/weave)
+* [Flannel](https://github.com/coreos/flannel)
+* [Canal](https://docs.projectcalico.org/getting-started/kubernetes/flannel/flannel)
+* [Calico](https://www.projectcalico.org/)
+
+##### Configuration
+
+**Optional**: You only have to export the KUBECONFIG environment variable.
+Otherwise, liqoctl will try to use the kubeconfig in kubectl default path (i.e. `${HOME}/.kube/config` )
+
+```bash
+export KUBECONFIG=/your/kubeconfig/path
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+
+#### Install
+
+Now, you can perform the proper Liqo installation on your cluster.
+
+{{< tabs >}}
+{{% tab name="K8s (Kubeadm)" %}}
+```bash
+liqoctl install --provider kubeadm
+```
+{{% /tab %}}
+{{% tab name="EKS" %}}
+
+{{% notice note %}}
+To run the following command, you must have permission to use the `eks:DescribeCluster` API action with the cluster you specify.
+{{% /notice %}}
+
+```bash
+liqoctl install --provider eks --eks.region=${EKS_CLUSTER_REGION} --eks.cluster-name=${EKS_CLUSTER_NAME} 
+```
+{{% /tab %}}
+{{% tab name="AKS" %}}
+```bash
+liqoctl install --provider aks --aks.resource-group-name ${AZURE_RESOURCE_GROUP} --aks.resource-name ${AZURE_RESOURCE_NAME} --aks.subscription-id ${AZURE_SUBSCRIPTION_ID}"
+```
+{{% /tab %}}
+{{% tab name="GKE" %}}
+```bash
+export KUBECONFIG=/your/kubeconfig/path
```

Comment:
Should be moved in the pre-requirements, for consistency with the other providers.